markerclustererplus/src/markerclusterer.js

```diff
@@ -120,31 +121,37 @@ ClusterIcon.prototype.onAdd = function () {
 
   this.div_ = document.createElement("div");
   this.div_.className = this.className_;
+  this.eventDiv_ = this.div_.cloneNode();
+  this.eventDiv_.className += ' event';
```

Comment:
Any particular reason for adding the "event" class name to the overlay? Shouldn't it be exactly the same class as the overlayLayer div?

Reply:
I'd like to have a way to differentiate the visible div, `div.classname` and the event div, `div.classname.event` . The same class configured by the user is on both divs, but event is added so that you can use a `:not` or other selector.

I guess I could use a more BEM friendly `div.classname.div.classname--event` if you'd prefer?

Reply:
The two divs should be absolutely identical, so I think it's best if the overlayMouseTarget div has exactly the same class as the overlayLayer div.

Reply:
Done